src/storage-client/src/client.rs

```diff
@@ -92,6 +92,10 @@ where
 }
 
 /// Commands related to the ingress and egress of collections.
+///
```

Comment:
You've been bitten by this?

Reply:
If you look at the reconciliation behavior:
https://github.com/MaterializeInc/materialize/blob/main/src/storage/src/storage_state.rs#L1003

It iterates over the set and only retains some of the operations, the others are "reconciled" away.